CHANGELOG.txt

```diff
@@ -0,0 +1,19 @@
+origin/8.x-4.x, 2019-02-20
+--------------------------
+- .
+- Code sniff cleanups.
+- Fixes tests.
+- Update typed_data to be a hard requirement for salesforce_mapping.
+- Merge branch '8.x-3.x' into 8.x-4.x.
+- WIP.
+- WIP EOD.
+- Debugging for deprecations.
+- WIP 4.x branch.
+- Add deprecations.
+- Merge branch '8.x-3.x' of bitbucket.org:messageagency/sf into auth_refactor_phase2.
+- Couple more updates to docs.
+- Code sniff cleanup.
+- Merge branch '8.x-3.x' of bitbucket.org:messageagency/sf into auth_refactor_phase1.
+- Merge branch '8.x-3.x' of github.com:messageagency/sfd8 into auth_refactor_phase1.
+- Update hook to update acess tokens.
+- Auth refactor, refactored. Phase 1 of ?.
```

Comment:
@ironsizide Git log is not particularly helpful.
A list of issues would be more useful.

There's a tool for this: https://www.drupal.org/project/grn

Reply:
Believe it or not, that's the tool I used to generate this. I imagine I didn't use the proper commit refs. Doesn't grn also depend on  the commit messages being in a particular format? https://www.drupal.org/node/52287

Reply:
Ah, yes, it's not GRN, it's the "release notes" tool from this project: https://github.com/mglaman/drupalorg-cli

Via https://glamanate.com/blog/creating-better-drupal-module-release-notes